### DISCUSSION TITLE
dom.serviceWorkers

### DISCUSSION THREAD

**Discussion by Jhedrick1023** [mozilla/policy-templates#883] [General]
Is there a way to manage dom.serviceWorkers using ADMX? I'm being asked to place a workaround for a SharePoint issue by changing dom.serviceWorkers.navigationPreload.enabled to False.

**Comment by mkaply** [comment]
Yes. All of the dom preferences can be set using Preferences

https://github.com/mozilla/policy-templates/blob/master/README.md#preferences

So you can do something like:

```
{
           "dom.serviceWorkers.navigationPreload.enabled": {
                "Value": false
            }
}
```

**Comment by mkaply** [comment]
You have me curious though. Is this a documented workaround somewhere?

And even more so, do you happen to use a local sharepoint server versus the cloud version? I desperately need to debug some sharepoint 2019 issues :).

**Comment by Jhedrick1023** [comment]
I'm not the person who found this workaround - but as of Firefox 99 we started experiencing issues with SharePoint Online not loading pages/sites properly. It was determined that the Experimental Feature of preloading service workers was enabled in this release. Once we manually disabled the feature, pages/sites started loading properly.

**Comment by mkaply** [comment]
Interesting. We haven't had any reported issues of this.

**Comment by Jhedrick1023** [comment]
Microsoft is aware of the issue as they released a Service Alert on it - and several people have posted on Mozilla Support forums for it as well.

Once Microsoft releases the fix, I can reverse the policy and see if the issue is corrected.
